typescript/nomad-deploy/src/core/CoreDeploy.ts

```diff
@@ -95,6 +96,26 @@ export class CoreDeploy extends Deploy<CoreContracts> {
     return configs;
   }
 
+  static buildSDK(local: CoreDeploy, remotes: CoreDeploy[]): NomadDomain {
+    return {
+      name: local.chain.name,
+      id: local.chain.domain,
+      paginate: {
+        from: local.config.fromBlock || 0,
+        blocks: local.chain.config.chunk || 2000,
+      },
+      home: local.contracts.home!.proxy.address,
+      replicas: remotes.map(remote => {
+        return {
+          domain: remote.chain.domain,
+          address: remote.contracts.replicas[local.chain.domain].proxy.address
+        }
+      }),
+      bridgeRouter: "n/a",
```

Comment:
make optional in the NomadDomain type and add handling to `BridgeContracts` for missing keys

Reply:
didn't address this one cuz the best way to do it would be for the bridge to read and then update the existing sdk deploy but that is more work than i want to deal with right now